src/__mocks__/MockComponent.jsx

```diff
@@ -0,0 +1,23 @@
+import useTheme from '../Theme'
+import {ThemeProvider} from '@mui/material/styles'
+import React, {createContext} from 'react'
+
+const ColorModeContext = createContext({
```

Comment:
Also here 2 empty lines.

Prefer MockComponent to be first code block since it matches filename, with ColorModeContext below.

Reply:
Corrected in ea796559bdadd91c4cfa3575fbccde1ef09c17ac.